control_plane/storcon_cli/src/main.rs

```diff
@@ -1043,7 +1043,8 @@ async fn main() -> anyhow::Result<()> {
                     sk.host,
                     format!("{}", sk.port),
                     format!("{}", sk.http_port),
-                    sk.availability_zone_id.to_string(),
+                    sk.availability_zone_id.clone(),
+                    String::from(sk.scheduling_policy),
```

Comment:
Call to `set_header` above should be updated too.

Reply:
done